_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual-steps.md

```diff
@@ -0,0 +1,137 @@
+---
+pagination_label: Deployment Walkthrough
+---
+
+# Manual steps
+
+Now that your infrastructure is up and running, there are some manual steps necessary before the resources meet the CIS AWS Benchmark recommendations.
+
+## 1. Revoke Gruntwork's access
+
+Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. The access is given through an IAM role called `GruntworkAccessRole` in each of the accounts through the `AdministratorAccess` policy. Use the `gruntwork` CLI to delete the IAM role and revoke access in each account by [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):
```

Comment:
```suggestion
Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. The access is given through an IAM role called `GruntworkAccountAccessRole` in each of the accounts through the `AdministratorAccess` policy. Use the `gruntwork` CLI to delete the IAM role and revoke access in each account by [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):
```

Reply:
We actually name that IAM role: `GruntworkAccountAccessRole` - which you can see here in the gruntwork CLI: https://github.com/gruntwork-io/gruntwork/blob/dc2a0f3ad75e1b371ab413270e7addc544b395b4/options/defaults.go#L9